feature-group-definitions/shadow-dom.yml

```
@@ -1,4 +1,5 @@
 name: Shadow DOM
+description: Shadow DOM is a part of Web Components. It allows for attaching encapsulated "shadow" DOM trees to the DOM. A shadow DOM tree is rendered separately from the main DOM, avoiding the component's scripts and styles to collide with other parts of the document.
```

This is one where I think a "X does Y" description isn't practical. I like this approach, just tweaking:

```suggestion
description: Shadow DOM allows you to attach encapsulated "shadow" DOM trees to elements. A shadow DOM tree is a separate component, isolated from the scripts and styles in other parts of the document. This is a part of Web Components.
```